sui_types/src/crypto.rs

```diff
@@ -70,6 +71,27 @@ impl<'de> Deserialize<'de> for KeyPair {
     }
 }
 
+impl FromStr for KeyPair {
+    type Err = anyhow::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        let value = Base64::decode_vec(s).map_err(|e| anyhow!("{}", e.to_string()))?;
+        let key = dalek::Keypair::from_bytes(&value).map_err(|e| anyhow!("{}", e.to_string()))?;
+        Ok(KeyPair {
+            key_pair: key,
+            public_key_cell: OnceCell::new(),
+        })
+    }
```

Comment:
Where would this be used?

Reply:
Has to parse from command line when someone inputs their key